packages/frontend/src/hooks/usePieChartConfig.ts

```diff
@@ -286,10 +285,9 @@ const usePieChartConfig: PieChartConfigFn = (
 
     const groupColorDefaults = useMemo(() => {
         return Object.fromEntries(
-            groupLabels.map((name, index) => [
-                name,
-                colorPalette[index % colorPalette.length],
-            ]),
+            groupLabels.map((name, index) => {
+                return [name, colorPalette[index % colorPalette.length]];
```

Comment:
This code will eventually be removed, once the feature flag is removed too - it's kept around to make it easy to switch between the flagged/not-flagged behavior.